uptane-standard.md

```diff
@@ -822,7 +833,8 @@ In order to perform full verification, an ECU SHALL perform the following steps:
 11. Verify that Targets metadata from the Director and Image repositories match. A primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the target metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
     1. Locate and download a Targets metadata file from the Image repository that contains an image with exactly the same file name listed in the Director metadata, following the procedure in {{resolve_delegations}}.
     2. Check that the Targets metadata from the Image repository matches the Targets metadata from the Director repository:
-        1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted image are the same in both sets of metadata.
+        1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted or encrypted image are the same in both sets of metadata.
+            - Please note, the primary is responsible for validating encrypted images and associated metadata. The target ECU (primary or secondary) is responsible for validating the unencrypted image and associated metadata.
```

Comment:
```suggestion
            - Note, the primary is responsible for validating encrypted images and associated metadata. The target ECU (primary or secondary) is responsible for validating the unencrypted image and associated metadata.
```

Reply:
Updated.

Also, allow me to respond to your question from lines 644&805 here. 

> "Are we on agreement on this? I think the understanding based on the last call was that we do not discuss ECUs that cannot do Uptane."

I agree this was the result of the conversation from the last call.
The way I interpreted this was to include this explicit detail, rather than the previous approach which made it seem more conditional for a secondary to perform verification.

Hope this clarification helps!

Reply:
Could you please make a quick fix? Please say `Note: ` instead of `Note, `

Reply:
Also remove the extra indentation, please. We can just include it at the end of the sentence preceding right before.